pkg/collector/helm3.go

```diff
@@ -30,12 +31,13 @@ func NewHelmV3Collector(opts *HelmV3Opts) (*HelmV3Collector, error) {
 	}
 
 	collector := &HelmV3Collector{
-		commonCollector: newCommonCollector("Helm v2"),
+		commonCollector: newCommonCollector("Helm v3"),
```

Comment:
Should the string be a const so you can refer to it in the test?

Reply:
:) Yeah, I was thinking about it... it would be nicer in the code, but then I'm not sure from the test perspective. As in the test then wouldn't catch if it gets changed to something undesirable. So 🤷‍♂️ ...

Happy to change if you think it's better, I really don't have a strong opinion here.

Reply:
Yeah I like the const. But yeah not a very strong opinion either way.

Reply:
Yeah, changed it, consts look nicer 👍